tools/reviewer/web/src/app/dashboard/reviews-panel/review/review-files/review-files.component.ts

```diff
@@ -1,25 +1,27 @@
-import { Files } from '@/shared';
 import { Component, Input } from '@angular/core';
 import { Router } from '@angular/router';
 
+import { File } from '@/shared/shell';
+
 @Component({
   selector: 'review-files',
   templateUrl: './review-files.component.html',
   styleUrls: ['./review-files.component.scss']
 })
 export class ReviewFilesComponent {
-  @Input() files: Files;
-  @Input() diffId;
+  isLoading: boolean = true;
+  @Input() fileList: File[];
+  @Input() diffId: number;
+
+  constructor(private router: Router) { }
 
-  constructor(private router: Router) {}
+  ngOnInit() {
+    this.isLoading = false;
+  }
 
   // Upon click on a file open a single file review page showing
   // code difference and comments
-  openFile(filePosition): void {
-    // Build a route path on the following format /diff/<diff number>/<path>?
-    // ls=<left snapshot number>&rs=<right snapshot number>
-    this.router.navigate(['diff/' + this.diffId + '/' + filePosition], {
-      queryParams: { ls: '1', rs: '3' }
-    });
+  openFile(filename: string): void {
+    this.router.navigate(['diff/' + this.diffId + '/' + filename]);
```

Comment:
let's keep the comments for URL building unless Ofer has advised otherwise that I'm not aware of.

Reply:
There's already comment about it:
https://github.com/google/startup-os/blob/WD1/tools/reviewer/web/src/app/dashboard/reviews-panel/diff/diff.component.ts#L59

Reply:
Sorry, I don't see a comment that says the URL should be of a specific format but maybe it's a discussion between you and Ofer. Anyway, it's of little importance.

Reply:
I don't see any necessity to add something else to url or maintain some specific url format.
Maybe we'll need to add commit data to it in the future. Exactly the question is raised in the link, which I send above.

Reply:
We'll definitely need to add commit data to it in the future and in the format described in the comment. That is exactly why we need to maintain a specific url format.

Reply:
Sorry, I don't follow.. 
I'm fine with the code as it is. Vadim removed a comment that explained the `this.router.navigate()` call. I think that's fine.
There wasn't any discussion around this AFAIR, and the format can change. It just needs to be reasonable and have all that's needed to be able to send the url to someone and he should open the same page.

Reply:
@oferb we talked about the URL will have a format including leftsnapshot and rightsnapshot number. We're discussing that comment. It is relevant to this discussion https://github.com/google/startup-os/pull/140/files#r198593942

Reply:
We don't use snapshots anymore.

Reply:
I forgot about that! So yes, we should have those parameters in the url, but they should point to commits, not snapshots. They could still be a number (doesn't have to be the commit sha).